### DISCUSSION TITLE
Welcome to DevSecOps Discussions!

### DISCUSSION THREAD

**Discussion by jdhaines** [aiaa-deic/devsecops#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our subcommittee. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other team members
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ba-napitts** [comment]
Checking access
